structs.go

```diff
@@ -1336,57 +1336,168 @@ const (
+	ErrCodeMessageCannotBeEditedDueToAnnouncementRateLimits                 = 20022
+	ErrCodeChannelHasHiTheWriteRateLimit                                    = 20028
+	ErrCodeTheWriteActionYouArePerformingOnTheServerHasHitTheWriteRateLimit = 20029
+	ErrCodeStageTopicContainsNotAllowedWordsForPublicStages                 = 20031
+	ErrCodeGuildPremiumSubscriptionLevelTooLow                              = 20035
+
+	ErrCodeMaximumGuildsReached                                    = 30001
+	ErrCodeMaximumFriendsReached                                   = 30002
+	ErrCodeMaximumPinsReached                                      = 30003
+	ErrCodeMaximumNumberOfRecipientsReached                        = 30004
+	ErrCodeMaximumGuildRolesReached                                = 30005
+	ErrCodeMaximumNumberOfWebhooksReached                          = 30007
+	ErrCodeMaximumNumberOfEmojisReached                            = 30008
+	ErrCodeTooManyReactions                                        = 30010
+	ErrCodeMaximumNumberOfGuildChannelsReached                     = 30013
+	ErrCodeMaximumNumberOfAttachmentsInAMessageReached             = 30015
+	ErrCodeMaximumNumberOfInvitesReached                           = 30016
+	ErrCodeMaximumNumberOfAnimatedEmojisReached                    = 30018
+	ErrCodeMaximumNumberOfSeverMembersReached                      = 30019
+	ErrCodeMaximumNumberOfGuildDiscoverySubcategoriesReached       = 30030
+	ErrCodeGuildAlreadyHasATemplate                                = 30031
+	ErrCodeMaximumNumberOfThreadParticipantsReached                = 30033
+	ErrCodeMaximumNumberOfBansForNonGuildMembersHaveBeenExceeded   = 30035
+	ErrCodeMaximumNumberOfBansFetchesHasBeenReached                = 30037
+	ErrCodeMaximumNumberOfUncompletedGuildScheduledEventsReached   = 30038
+	ErrCodeMaximumNumberOfStickersReached                          = 30039
+	ErrCodeMaximumNumberOfPruneRequestsHasBeenReached              = 30040
+	ErrCodeMaximumNumberOfGuildWidgetSettingsUpdatesHasBeenReached = 30042
+	ErrCodeMaximumNumberOfEditsToMessagesOlderThanOneHourReached   = 30046
+
+	ErrCodeUnauthorized                           = 40001
+	ErrCodeActionRequiredVerifiedAccount          = 40002
+	ErrCodeOpeningDirectMessagesTooFast           = 40003
+	ErrCodeSendMessagesHasBeenTemporarilyDisabled = 40004
+	ErrCodeRequestEntityTooLarge                  = 40005
+	ErrCodeFeatureTemporarilyDisabledServerSide   = 40006
+	ErrCodeUserIsBannedFromThisGuild              = 40007
+	ErrCodeTargetIsNotConnectedToVoice            = 40032
+	ErrCodeMessageAlreadyCrossposted              = 40033
+	ErrCodeAnApplicationWithThatNameAlreadyExists = 40041
+
+	ErrCodeMissingAccess                                                = 50001
+	ErrCodeInvalidAccountType                                           = 50002
+	ErrCodeCannotExecuteActionOnDMChannel                               = 50003
+	ErrCodeEmbedDisabled                                                = 50004
```

Comment:
Maybe it would be better to name it `ErrCodeGuildWidgetDisabled`, to avoid confusion with message embeds